### ISSUE TITLE
Data normalization tutorial

### ISSUE THREAD

**Issue by thinhphamtechvico2021** [aliprf/ASMNet#3]
Hi creator,

Could you please provide a tutorial or a script for data normalization? I have been struggling for a while but couldn't figure out how to do it.

**Comment by aliprf** [comment]
Hi, 
thank you for being interested in the repo. Data augmentation is not part of this repo, since the goal is to describe the ASM loss function. The following link may help you:
https://www.tensorflow.org/tutorials/images/data_augmentation
